### DISCUSSION TITLE
Contact Form

### DISCUSSION THREAD

**Discussion by aman9267** [webexpe13/blog-template-using-nextjs-typescript-tailwindcss#6] [Q&A] [answered]
Hi Sir,
Here is my task detail
Task Details 
Once done create a contact page to submit data in backend by calling API exposed from backend.
 Contact Us Fields : Name , Mobile , Email , Message
Create a view page  by fetching data for contact us data.

I have created a form and the data is stored in console log. So sir tell me is there any api to submit data or fetch data. If there is any slim please send me the api url or else.

**Comment by webexpe13** [comment, marked as answer]
Hi Aman,
To send and fetch data you will need an API, for your form submit data you will have to do a POST request on an API. 
Check out this article [https://webexpe.com/blog/how-to-make-api-call-in-react-using-fetch-and-axios/](https://webexpe.com/blog/how-to-make-api-call-in-react-using-fetch-and-axios/) this will help you understand how to make API calls in react.
